cpu/esp32/esp-idf/include/driver/gpio.h

```diff
@@ -0,0 +1,16 @@
+#ifndef DRIVER_GPIO_H
```

Comment:
The CI is upset about missing copyright and `extern "C" ` boilerplates

Reply:
Other than the missing boilerplates, this looks good to me. Please squash at will.